Hudl.Mjolnir/Hudl.Mjolnir.csproj

```diff
@@ -109,6 +120,7 @@
     <None Include="Hudl.Mjolnir.nuspec" />
     <None Include="packages.config" />
   </ItemGroup>
+  <ItemGroup />
```

Comment:
Ah, but I do! Actually, no. I don't. :)

I don't think these matter too much - I think VS creates them when new files are added but then forgets do delete them if you move the files around.